### PR TITLE
Updated Project Name

### PR DESCRIPTION
**Pull request by YAS-CODE**

### CODE REVIEW

File: Demo/Properties/launchSettings.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      "commandName": "IISExpress",
+      "launchBrowser": true,
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    },
+    "Demo": {
+      "commandName": "Project",
+      "launchBrowser": true,
+      "applicationUrl": "https://localhost:5001;http://localhost:5000",
```
Comment (AlexeyAnshakov) [inline Demo/Properties/launchSettings.json:21]: Are you sure about using localhost here?

File: Demo/obj/Debug/netcoreapp3.0/Razor/Pages/Error.cshtml.g.cs [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+#line 10 "C:\Users\Yas\source\repos\Demo\WebrunsDemo\Pages\Error.cshtml"
+ if (Model.ShowRequestId)
+{
+
+#line default
+#line hidden
+#nullable disable
+            WriteLiteral("    <p>\r\n        <strong>Request ID:</strong> <code>");
+#nullable restore
+#line 13 "C:\Users\Yas\source\repos\Demo\WebrunsDemo\Pages\Error.cshtml"
```
Comment (AlexeyAnshakov) [inline Demo/obj/Debug/netcoreapp3.0/Razor/Pages/Error.cshtml.g.cs]: still a lot of wrong named lines like this. Use webRunes or avoid it at all

File: Demo/obj/Debug/netcoreapp3.0/WebrunsDemo.AssemblyInfo.cs [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+using System;
+using System.Reflection;
+
+[assembly: System.Reflection.AssemblyCompanyAttribute("WebrunsDemo")]
+[assembly: System.Reflection.AssemblyConfigurationAttribute("Debug")]
+[assembly: System.Reflection.AssemblyFileVersionAttribute("1.0.0.0")]
+[assembly: System.Reflection.AssemblyInformationalVersionAttribute("1.0.0")]
+[assembly: System.Reflection.AssemblyProductAttribute("WebrunsDemo")]
+[assembly: System.Reflection.AssemblyTitleAttribute("WebrunsDemo")]
```
Comment (AlexeyAnshakov) [inline Demo/obj/Debug/netcoreapp3.0/WebrunsDemo.AssemblyInfo.cs]: Again

File: Demo/obj/Debug/netcoreapp3.0/Demo.csproj.FileListAbsolute.txt [hunk 4]
```diff
@@ -82,3 +82,45 @@ C:\Users\Yas\source\repos\Demo\Demo\obj\Debug\netcoreapp3.0\Demo.Views.pdb
 C:\Users\Yas\source\repos\Demo\Demo\obj\Debug\netcoreapp3.0\Demo.csproj.CopyComplete
```
Comment (AlexeyAnshakov) [inline Demo/obj/Debug/netcoreapp3.0/Demo.csproj.FileListAbsolute.txt]: Still see debug files

File: Demo/obj/Demo.csproj.nuget.dgspec.json [hunk 5]
```diff
@@ -1,17 +1,17 @@
 {
   "format": 1,
   "restore": {
-    "C:\\Users\\Yas\\source\\repos\\WebRunsDemo\\Demo\\Demo.csproj": {}
+    "C:\\Users\\Yas\\source\\repos\\Testbed-GatewayServer\\Demo\\Demo.csproj": {}
```
Comment (AlexeyAnshakov) [inline Demo/obj/Demo.csproj.nuget.dgspec.json:4]: We don't need files related to localhost project

File: Demo/obj/WebrunsDemo.csproj.nuget.dgspec.json [hunk 6]
```diff
@@ -0,0 +1,73 @@
+{
+  "format": 1,
+  "restore": {
+    "C:\\Users\\Yas\\source\\repos\\Demo\\WebrunsDemo\\WebrunsDemo.csproj": {}
```
Comment (AlexeyAnshakov) [inline Demo/obj/WebrunsDemo.csproj.nuget.dgspec.json]: WebrunsDemo again

File: Demo/obj/WebrunsDemo.csproj.nuget.g.props [hunk 7]
```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
```
Comment (AlexeyAnshakov) [inline Demo/obj/WebrunsDemo.csproj.nuget.g.props]: WebrunsDemo

File: Demo/obj/Debug/netcoreapp3.0/Razor/Pages/Error.cshtml.g.cs (hunk 2 above)
Comment (YAS-CODE) [reply Demo/obj/Debug/netcoreapp3.0/Razor/Pages/Error.cshtml.g.cs]: This File does not exist in our future commit

File: Demo/obj/Debug/netcoreapp3.0/WebrunsDemo.AssemblyInfo.cs (hunk 3 above)
Comment (YAS-CODE) [reply Demo/obj/Debug/netcoreapp3.0/WebrunsDemo.AssemblyInfo.cs]: This File does not exist in our future commit

File: Demo/obj/Debug/netcoreapp3.0/Demo.csproj.FileListAbsolute.txt (hunk 4 above)
Comment (YAS-CODE) [reply Demo/obj/Debug/netcoreapp3.0/Demo.csproj.FileListAbsolute.txt]: This File does not exist in our future commit

File: Demo/obj/WebrunsDemo.csproj.nuget.g.props (hunk 7 above)
Comment (YAS-CODE) [reply Demo/obj/WebrunsDemo.csproj.nuget.g.props]: This File does not exist in our future commit

File: Demo/obj/WebrunsDemo.csproj.nuget.dgspec.json (hunk 6 above)
Comment (YAS-CODE) [reply Demo/obj/WebrunsDemo.csproj.nuget.dgspec.json]: This File does not exist in our future commit

File: Demo/Properties/launchSettings.json (hunk 1 above)
Comment (YAS-CODE) [reply Demo/Properties/launchSettings.json:21]: For now its fine with localhost

**Pull request merged**